force-app/main/default/classes/ClassEnrollmentTriggerHelperTests.cls

```diff
@@ -0,0 +1,48 @@
+@isTest
+public with sharing class ClassEnrollmentTriggerHelperTests {
+    @TestSetup
+    static void makeData() {
+        SchoolDataCreator.createData();
+        
+        // Class Enrollment creator
+        ClassEnrollment__c ce = new ClassEnrollment__c(
+            Class__c = [SELECT Id FROM Class__c][0].Id, 
```

Comment:
Just LIMIT 1. No need to store all of the Class__c records. That could be a lot in a production environment.